ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -158,6 +158,24 @@ Item {
               color: Theme.palette.directColor1
           }
 
+          MouseArea {
+              property bool accountPopupOpened: false
+
+              anchors.fill: parent
+              anchors.margins: -10
+              cursorShape: Qt.PointingHandCursor
+              hoverEnabled: true
+              onEntered: {
+                  accountPopupOpened = accountsPopup.opened
```

Comment:
Is that intended, to have the popup open on mouse hover?

Reply:
It does not open but just gets popup state. If it is open already we do nothing on click, else call function which opens it.

Reply:
Maybe I need better naming like isAcountPopupOpened ?

Reply:
Nah, not about naming, I fail to see the need for the bool property :)

Reply:
Got it :)